apps/cowswap-frontend/src/common/hooks/useBannerVisibility.ts

```diff
@@ -0,0 +1,23 @@
+import { useState, useEffect } from 'react';
+
+export enum LSKeys {
```

Comment:
Why is it called LSKeys? What does `LS` stands for?

Reply:
LocalStorage

Reply:
Got it!

Reply:
Added inline comment to clarify. Just think it benefits from having a shorter name :)